tests/unit/APITest.js

```diff
@@ -168,8 +168,7 @@ describe('APITests', () => {
                     expect(PersistedRequests.getAll()).toEqual([expect.objectContaining({command: 'mock command', data: expect.objectContaining({param2: 'value2'})})]);
 
                     // We need to advance past the request throttle back off timer because the request won't be retried until then
-                    jest.advanceTimersByTime(CONST.NETWORK.MAX_RANDOM_RETRY_WAIT_TIME_MS);
-                    return waitForPromisesToResolve();
+                    return new Promise((resolve) => setTimeout(resolve, 1000)).then(waitForPromisesToResolve);
```

Comment:
This isn't going to actually take 1s to resolve right?

Reply:
True! Fixed!